:recycle: Accepts new path parameters and updates tests (#1784)
ministryofjustice/analytical-platform: containers/daap-presigned-url/CHANGELOG.md

## [Unreleased]

## [1.2.4] 2023-10-03

### Changed

> - removed datebase and table querystring parameters

Reviewer: ```suggestion
- removed database and table querystring parameters
```

> - created date-product-name & table-name variables from event body pathParameters

Reviewer: ```suggestion
- created data-product-name & table-name variables from event body pathParameters
```

- created body variable from event.body json string

> - replaced refferences to datebase with data-product-name

Reviewer: ```suggestion
- replaced references to database with data-product-name
```

> - replaced refferences to table with table-name

Reviewer: ```suggestion
- replaced references to table with table-name
```
